packages/p2p/stores/advertiser-page-store.js

```diff
@@ -36,22 +37,21 @@ export default class AdvertiserPageStore {
         height_constants.tabs,
     ];
     item_height = 56;
-    props = {};
 
     get account_currency() {
-        return this.props?.selected_advert.account_currency;
+        return this.buy_sell_store.selected_ad_state.account_currency;
```

Comment:
Should `AdvertiserPageStore` maintain its own state rather than getting it from `BuySellStore`? 😕